src/detectors/ECTOF/ECTOF.cc

```diff
@@ -54,10 +52,10 @@ void InitPlugin(JApplication* app) {
   app->Add(new JOmniFactoryGeneratorT<SiliconChargeSharing_factory>(
       "TOFEndcapSharedHits", {"TOFEndcapHits"}, {"TOFEndcapSharedHits"},
       {
-          .sigma_mode     = SiliconChargeSharingConfig::ESigmaMode::abs,
-          .sigma_sharingx = 0.1 * dd4hep::cm,
-          .sigma_sharingy = 0.1 * dd4hep::cm,
-          .min_edep       = 0.0 * edm4eic::unit::GeV,
+          .sigma_mode     = SiliconChargeSharingConfig::ESigmaMode::rel,
```

Comment:
```suggestion
          .sigma_mode     = SiliconChargeSharingConfig::ESigmaMode::abs,
```
You can't use units and `rel` mode.